compiler/ballerina-parser/src/test/resources/statements/transaction-stmt/transaction_stmt_source_09.bal

```diff
@@ -1,34 +1,34 @@
 function foo() {
     transaction {
-    } on fail error error err {
-        io:println(err);
-    }
-
-    transaction {
-        int a = 5;
-    } on fail error error err {
-        io:println(err);
-    }
+    } on fail error error err{
+io:println(err);
+}
 
-    transaction {
         transaction {
-            transaction {
-                string b;
-            }
-        }
-    } on fail error error err {
-        io:println(err);
-    }
+            int a = 5;
+        } on fail error error err{
+io:println(err);
+}
 
-    transaction {
-        transaction {
             transaction {
-                string b;
-            } on fail error error err {
-                io:println(err);
-            }
-        }
-    } on fail error error err {
-        io:println(err);
-    }
+                transaction {
+                    transaction {
+                        string b;
+                    }
+                }
+            } on fail error error err{
+io:println(err);
+}
+
+                transaction {
+                    transaction {
+                        transaction {
+                            string b;
+                        } on fail error error err{
+io:println(err);
```

Comment:
Not easy to understand the test case. Since we have skipped this in the formatter let's keep this test without formatting.